src/applications/maniphest/config/PhabricatorManiphestConfigOptions.php

```diff
@@ -247,6 +247,80 @@ public function getOptions() {
     $json = new PhutilJSON();
     $status_example = $json->encodeFormatted($status_example);
 
+    $category_type = 'maniphest.categories';
+    $category_defaults = array();
+
+    $category_description = $this->deformat(pht(<<<EOTEXT
```

Comment:
Assuming the coding practices are followed for this repo as I don't see the separation of static resources in the codebase.